### ISSUE TITLE
torch.jit.script works for x*sigmoid(x) but not for x*sin(x)

### ISSUE THREAD

**Issue by adityaiitb** [pytorch/pytorch#35408]
cc: @ptrblck 

## 🐛 Bug

In the following example, I observe that `x * sigmoid(x)` is JITted as expected. In the first call to `foo`, I see 1 GPU kernel for sigmoid and 1 for mul. In the second call, I see 1 call to `kernel_0` which is the JITted kernel. However, if I replace `sigmoid` wtih `sin`, seemingly another simple pointwise operation, then the JIT does not seem to be working and I observe 4 kernels (2 for sin and 2 for mul). However, if I use the flags

```python
torch._C._jit_set_profiling_mode(False)
torch._C._jit_set_profiling_executor(False)
```
then I see `kernel_0` being invoked twice as expected.

## To Reproduce

```python
import torch
@torch.jit.script
def foo(x):
    return  x * torch.sigmoid(x)
    #return  x * torch.sin(x)

x = torch.rand(2048,2048).cuda()
y = foo(x)
y = foo(x)
```
## Steps to reproduce the behavior:

Run the above code as `nvprof python foo.py` to see the list of kernels.

## Expected behavior

I would expect `x * sin(x)` to be JITted as well.

## Environment
```vim
PyTorch version: 1.4.0a0+a5b4d78
Is debug build: No
CUDA used to build PyTorch: 10.2

OS: Ubuntu 18.04.4 LTS
GCC version: (Ubuntu 7.4.0-1ubuntu1~18.04.1) 7.4.0
CMake version: version 3.14.0

Python version: 3.6
Is CUDA available: Yes
CUDA runtime version: 10.2.89
GPU models and configuration: 
GPU 0: Tesla V100-SXM2-16GB
GPU 1: Tesla V100-SXM2-16GB
GPU 2: Tesla V100-SXM2-16GB
GPU 3: Tesla V100-SXM2-16GB
GPU 4: Tesla V100-SXM2-16GB
GPU 5: Tesla V100-SXM2-16GB
GPU 6: Tesla V100-SXM2-16GB
GPU 7: Tesla V100-SXM2-16GB

Nvidia driver version: 440.33.01
cuDNN version: /usr/lib/x86_64-linux-gnu/libcudnn.so.7.6.5

Versions of relevant libraries:
[pip] msgpack-numpy==0.4.3.2
[pip] numpy==1.17.2
[pip] pytorch-transformers==1.1.0
[pip] torch==1.4.0a0+a5b4d78
[pip] torch-dct==0.1.5
[pip] torchio==0.13.18
[pip] torchtext==0.4.0
[pip] torchvision==0.5.0a0
[conda] magma-cuda101             2.5.2                         1    local
[conda] mkl                               2020.0                      166  
[conda] mkl-include                  2020.0                      166  
[conda] nomkl                                3.0                           0  
[conda] pytorch-transformers      1.1.0                    pypi_0    pypi
[conda] torch                     1.4.0a0+a5b4d78          pypi_0    pypi
[conda] torchtext                          0.4.0                    pypi_0    pypi
[conda] torchvision                      0.5.0a0                  pypi_0    pypi
```

**Comment by suo** [comment]
cc @Krovatkin this seems to be a regression related to the profiling executor; can you comment?

**Comment by janeyx99** [comment]
TorchScript has been deprecated in favor of torch.export as of 2.11. https://pytorch.org/blog/pytorch-2-11-release-blog/ Closing this issue as we are not further developing on it--please ping if you believe this issue was erroneously closed.
